.dotnet.azure/sdk/openai/Azure.AI.OpenAI/src/Custom/AzureOpenAIClient.cs

```diff
@@ -122,7 +122,7 @@ public AzureOpenAIClient(Uri endpoint, ApiKeyCredential credential, AzureOpenAIC
     /// <param name="credential"> The API key to authenticate with the service. </param>
     /// <param name="options"> The options to configure the client. </param>
     public AzureOpenAIClient(Uri endpoint, AzureKeyCredential credential, AzureOpenAIClientOptions options)
-        : this(CreatePipeline(credential?.Key, options), endpoint, options)
+        : this(CreatePipeline(new ApiKeyCredential(credential?.Key), options), endpoint, options)
```

Comment:
nit: IDK if it matters or if it the right place to do this, but in the arch board yesterday IIRC they asked to remove this constructor so you could consider doing that here.  Also, a follow-up PR might be fine, I don't feel strongly.

Reply:
Given the size of this PR right now, I want to keep it laser focused on just the minimum updates needed for code to compile and run properly with System.ClientModel 1.1.0. I think that any API changes unrelated to SCM 1.1.0 should go in a separate PR.